pkg/admission/admission_test.go

```diff
@@ -138,16 +139,15 @@ func TestMutateNonStringsToStrings(t *testing.T) {
 
 	// Apply patch to original request
 	patchObj, err := jsonpatch.DecodePatch(resp.Response.Patch)
-	if err != nil {
-		t.Fatal(err, "Expected a valid patch")
-	}
+	require.NoError(t, err)
+
 	rev := v1.AdmissionReview{}
 	deserializer.Decode(golden.Get(t, "nonStringsInLabelsAnnotations.golden"), nil, &rev)
 	rev.Request.Object.Raw, err = patchObj.Apply(rev.Request.Object.Raw)
 	if err != nil {
 		fmt.Println(string(resp.Response.Patch))
-		t.Fatal(err, "Expected to successfully apply patch")
 	}
+	require.NoError(t, err)
```

Comment:
What do you think about using `NoErrorf` to just attach the message to remove the if block?

```go
	rev.Request.Object.Raw, err = patchObj.Apply(rev.Request.Object.Raw)
	require.NoErrorf(t, err, string(resp.Response.Patch))
```

Reply:
Yeah first I didn't think of it but in my other PR I made a change like this. Let me fix this.